app/views/components/docs/hide-this-page-banner.yml

```diff
@@ -1,4 +1,4 @@
-name: Escape link
+name: Hide this page banner
 description: Gives users a quick exit from a website
 body: |
   When JavaScript is disabled the link will open in a new tab so the perpetrator won’t be able to press back.
```

Comment:
I only just noticed this now and sorry I know it's not a change you introduced.

We should change the wording here so we don't use the phrase perpetrator as this seems odd in the context.

```
name: Hide this page banner
description: Provides a button for vulnerable users to switch to a neutral website
body: |
  This feature is intended for users who may be reading or entering sensitive information that they may quickly want to navigate away from should a third party be in their vicinity.
  Without JavaScript clicking the button will open a link in a new tab, this is so the current page in the viewport doesn't have a browser history.
  With JavaScript clicking the button will, in addition to the above, change the previous tab to the same link so the page is no longer open on the device.
```